packages/dso-toolkit/src/styles/dso.scss

```diff
@@ -1,10 +1,10 @@
 @import "mixins/mixins";
 @import "mixins/extends";
-@import "mixins/font-asap";
+// @import "mixins/font-asap"; -> Moved to @dso-toolkit/assets/fonts
 @import "mixins/full-width-row";
```

Comment:
Ik krijg een foutmelding in  font-asap.scss:
![image](https://user-images.githubusercontent.com/25430835/101614302-12d70780-3a0d-11eb-89f5-fe1536d2fcd5.png)